pkg/login/social/github_oauth.go

```diff
@@ -329,11 +329,14 @@ func convertToGroupList(t []GithubTeam) []string {
 	return groups
 }
 
-func mustInts(s []string) ([]int, error) {
+func mustInts(s []string, returnEmptyOnError bool) ([]int, error) {
```

Comment:
Nit. This extra argument is a bit weird given there's only one consumer of this so always true.

Reply:
Yeah, I think we can swallow the the error for now and add a TODO